tests/test_imageset.py

```diff
@@ -5,18 +5,42 @@
 
 import pytest
 import six.moves.cPickle as pickle
+from mock import patch
 
 from scitbx.array_family import flex
 
+import dxtbx.format.FormatHDF5SaclaMPCCD
 import dxtbx.format.image
 import dxtbx.format.Registry
 import dxtbx.tests.imagelist
 from dxtbx.format.FormatCBFMiniPilatus import FormatCBFMiniPilatus as FormatClass
 from dxtbx.imageset import ExternalLookup, ImageSequence, ImageSetData, ImageSetFactory
 from dxtbx.model import Beam, Detector, Panel
+from dxtbx.model.beam import BeamFactory
 from dxtbx.model.experiment_list import ExperimentListFactory
 
 
+def test_single_file_indices(dials_regression):
+    def dummy_beam():
+        return BeamFactory.simple(1.0)
+
+    with patch.object(
+        dxtbx.format.FormatHDF5SaclaMPCCD.FormatHDF5SaclaMPCCD,
+        "_beam",
+        side_effect=dummy_beam,
+    ) as obj:
+        filename = os.path.join(
+            dials_regression,
+            "image_examples",
+            "SACLA_MPCCD_Cheetah",
+            "run266702-0-subset.h5",
+        )
+        format_class = dxtbx.format.Registry.get_format_class_for_file(filename)
+        _ = format_class.get_imageset([filename], single_file_indices=[1])
```

Comment:
Does this actually need to be assigned to anything?
```suggestion
        format_class.get_imageset([filename], single_file_indices=[1])
```

Reply:
Nope, it's a copy/paste artifact